handlers/graph.go

```diff
@@ -212,13 +212,14 @@ func populateTrafficMap(trafficMap graph.TrafficMap, vector *model.Vector, o opt
 		// destination proxy.  But we still want to record the request in some
 		// way. So, depending on the graph type and labeling, assign the destination
 		// service name (i.e. the requested service) to the most applicable destination field.
+		// note: this code is duplicated in response_time.go for any changes/fixes
 		if destApp == graph.UnknownApp {
```

Comment:
shouldn't this check if destWl is unknown? see earlier comment

Reply:
Right, same problem, see comment about duplicated code :)

Reply:
can refactor this into a utility function somewhere and reuse so we don't duplicate bugs... er duplicate code? :)

Reply:
Probably, I actually stared at this for a while last night but couldn't decide on a good place to put the shared code. So you get this instead.  Maybe I'll move it later but not now.